src/components/navBar/index.tsx

```diff
@@ -76,7 +73,7 @@ const NavBar = () => {
                 </ul>
             </div>
             <div className={styles.userProfile}>
-                {!isLoggedIn ? (
+                {!isSuccess ? (
```

Comment:
is success returned by rtk will only return if fetch is successful or not, it wont return proper response in case the data comes in blank, because the fetch was successful.

Reply:
IMO the data will not come as an empty object because if the response in with in 200 range then only isSuccess value is truthy & in that case the user data exists in database & if the user data exists in database then the response is never null or empty, maybe the user has incomplete user details in this case